README.md

```
@@ -1,337 +1,203 @@
+├── models/                    # Modelos de dados (Entidades)
+│   ├── character.dart        # Modelo Character
+│   ├── location.dart         # Modelo Location
+│   └── episode.dart          # Modelo Episode
+├── repositories/             # Camada de acesso a dados
+│   ├── base_repository.dart  # Interface base para todos os repos
+│   ├── character_repository.dart
+│   ├── location_repository.dart
+│   └── episode_repository.dart
+└── main.dart                 # Ponto de entrada da aplicação
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)